src/robotide/editor/kweditor.py

```diff
@@ -728,13 +730,17 @@ def Clone(self):
 class ChooseUsageSearchStringDialog(wx.Dialog):
 
     def __init__(self, cellvalue):
+        from collections import OrderedDict
         wx.Dialog.__init__(self, None, wx.ID_ANY, "Find Where Used",
                            style=wx.DEFAULT_DIALOG_STYLE)
         self.caption = "Please select what you want to check for usage"
-        variables = utils.find_variable_basenames(cellvalue)
+        variables = variablematcher.find_variable_basenames(cellvalue)
+        variables = variablematcher.find_unique(variables)
```

Comment:
I'd prefer to chain these function calls instead of reassigning a variable with the same name.